### PR TITLE
Commiting changes made to css file for e10s settings width issues

### PR DESCRIPTION
**Pull request by ab0092**
I have added a new style section for the e10s setting width issues. The changes can be viewed from https://ab0092.github.io/telemetry-dashboard. Waiting for a review and response.

### CODE REVIEW

**Comment by chutten** [comment]
Thank you for your patch! Unfortunately, it makes the first dropdown (the one with the probe names in it) very difficult to read. We will need either a more targeted piece of CSS or a different approach.

**Comment by ab0092** [comment]
I have tweaked the css part a bit. Can u please check and confirm if this is gonna work

File: new-pipeline/style/dashboards.css [hunk 1]
```diff
@@ -43,6 +43,13 @@
     z-index: 1;
     top: 5px;
 }
+
+/** Max Width Setting For e10s **/
+.btn-group ul
+{
+    max-width: 40vw !important;
```
Comment (chutten) [inline new-pipeline/style/dashboards.css]: Can you target just e10s' ul? You should be able to use adjacent sibling selectors to find the `ul` of the sibling of `#filter-e10s`

Comment (ab0092) [reply new-pipeline/style/dashboards.css]: I have added the e10s specific css

**Comment by chutten** [comment]
Looks good! Can you squash your commits together before we get this merged?

**Comment by ab0092** [comment]
I am sorry chris but i missed u on this one i am very new to github so could you please guide me on how to go about my next step

**Comment by chutten** [comment]
On your local machine, `git rebase -i HEAD~3` should supply you with an editor that will allow you to squash, reorder, and otherwise mess with your commits. You want to squash all of your commits together into one. You'll get the opportunity to change the commit message. Maybe chose something like "bug 1396890 - Restrict the width of the e10s dropdown".

Then you'll need to force-push (pass `-f` to your `git push`) this to your `gh-pages` branch to overwrite the commits that are currently there. If you try to push without forcing, `git` will complain that it isn't a fast-forward change.

Then I'll merge it from here!

**Comment by ab0092** [comment]
Finally succeeded in merging all the commits. Thanks Chris.

**Comment by chutten** [comment]
Thanks! I have merged the changes. They should be picked up on production in the next few minutes.

If you are interested in writing more descriptive git commit messages, may I recommend you read [Chris Beams' post on commit messages](https://chris.beams.io/posts/git-commit/)? It really helps me understand what is useful to have in a commit message even for my own projects :)

**Comment by ab0092** [comment]
I will definitely go through the post and sharpen my commit messaging. Thank you all your assistance and help, i look forward to working with you again. I would like to continue working on some more issues or new work so i would be glad if you could assign me some new tasks. Thanks!

**Comment by chutten** [comment]
Please take a look at the Issues in this project and continue looking at the bugs in the bugzilla component for any that you might find interesting. If you're looking for ideas, #328 is a really simple one and #312 would be really helpful.

**Pull request merged**